### PR TITLE
Create a tab with all orders

### PR DESCRIPTION
**Pull request by apmasell**
In addition to the existing unfulfilled orders

### CODE REVIEW

**Comment by tonydebat** [comment]
After migrating to the production db I get the following error:

```
08-Aug-2016 17:18:52.098 SEVERE [http-nio-8080-exec-5] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [miso] in context with path [] threw exception [Request processing failed; nested exception is org.hibernate.exception.SQLGrammarException: could not extract ResultSet] with root cause
 com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Expression #5 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'miso.status.lastUpdated' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
        at com.mysql.jdbc.Util.getInstance(Util.java:387)
        at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
        at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
        at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
        at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2478)
        at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2625)
        at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2551)
        at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1861)
        at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1962)
        at org.apache.tomcat.dbcp.dbcp2.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:82)
        at org.apache.tomcat.dbcp.dbcp2.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:82)
        at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.extract(ResultSetReturnImpl.java:79)
        at org.hibernate.loader.Loader.getResultSet(Loader.java:2062)
        at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1859)
        at org.hibernate.loader.Loader.executeQueryStatement(Loader.java:1838)
        at org.hibernate.loader.Loader.doQuery(Loader.java:906)
        at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:348)
        at org.hibernate.loader.Loader.doList(Loader.java:2550)
        at org.hibernate.loader.Loader.doList(Loader.java:2536)
        at org.hibernate.loader.Loader.listIgnoreQueryCache(Loader.java:2366)
        at org.hibernate.loader.Loader.list(Loader.java:2361)
        at org.hibernate.loader.hql.QueryLoader.list(QueryLoader.java:495)
        at org.hibernate.hql.internal.ast.QueryTranslatorImpl.list(QueryTranslatorImpl.java:357)
        at org.hibernate.engine.query.spi.HQLQueryPlan.performList(HQLQueryPlan.java:198)
        at org.hibernate.internal.SessionImpl.list(SessionImpl.java:1230)
        at org.hibernate.internal.QueryImpl.list(QueryImpl.java:101)
        at uk.ac.bbsrc.tgac.miso.persistence.impl.HibernatePoolOrderCompletionDao.list(HibernatePoolOrderCompletionDao.java:61)
```

**Comment by apmasell** [comment]
@tonydebat I fixed that `GROUP BY` error.

**Comment by tonydebat** [comment]
Thanks, I knew something was wrong, but wasn't sure what.

**Comment by apmasell** [comment]
MySQL tends to be very sloppy about `GROUP BY` clauses. I guess you have the strictness check on and I don't.

**Comment by tonydebat** [comment]
👍

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/ListPoolOrdersController.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    ArrayNode headings = result.putArray("headings");
+    for (HealthType health : healths) {
+      headings.add(health.toString());
+    }
+    result.put("htmlElement", htmlName);
+    result.put("humanName", humanName);
+    return result;
+  }
+
+  @RequestMapping("/poolorders/outstanding")
```
Comment (callunity) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/ListPoolOrdersController.java]: Does it make sense to map this to `/outstanding` when it lists all of them?

**Comment by callunity** [comment]
:+1:

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/ListPoolOrdersController.java (hunk 1 above)
Comment (apmasell) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/ListPoolOrdersController.java]: Fixed.

**Pull request merged**